fineract-core/src/main/java/org/apache/fineract/commands/service/CommandWrapperBuilder.java

```diff
@@ -3930,4 +3930,44 @@ public CommandWrapperBuilder updateClientClassificationLimitMapping(final Long i
         this.href = "/savings/transactionlimits/classificationmapping/" + id;
         return this;
     }
+
```

Comment:
This change should be moved to the custom module. See how I have done it https://github.com/fiterpaystack/fineract/pull/43/files and maybe once that PR is merged you can reuse the same class.